production/ksonnet/loki/shipper.libsonnet

```diff
@@ -62,6 +62,7 @@
     container.mixin.readinessProbe.httpGet.withPort($._config.http_listen_port) +
     container.mixin.readinessProbe.withTimeoutSeconds(1) +
     k.util.resourcesRequests('4', '2Gi') +
+    k.util.resourcesLimits(null, '32Gi') +
```

Comment:
I'd suggest setting this 2x requests, and adding a note in the release notes for folks to set this limit to a figure appropriate for their env.

`32Gi` may not be appropriate in all cases.